src/api/python/setup.py.in

```diff
@@ -0,0 +1,21 @@
+from distutils.core import setup
+from distutils.command.clean import clean
+from distutils.command.install import install
+
+class Pycvc4Install(install):
+
+    # Calls the default run command, then deletes the build area
+    # (equivalent to "setup clean --all").
+    def run(self):
+        install.run(self)
+        c = clean(self.distribution)
+        c.all = True
+        c.finalize_options()
+        c.run()
+
+setup(name='pycvc4',
+      version='${CVC4_MAJOR}.${CVC4_MINOR}.${CVC4_RELEASE}',
+      packages=['pycvc4'],
+      package_dir={'pycvc4': '${CMAKE_CURRENT_BINARY_DIR}/pycvc4'},
+      package_data={'pycvc4': ['pycvc4.so']},
+      cmdclass={'install': Pycvc4Install})
```

Comment:
```suggestion
      cmdclass={'install': PyCVC4Install})
```

Reply:
I messed up the diff with my previous commit so I made the change here instead: ba6fa45bf7478f40768780cc7d4d5f829d3ac4ca